FingerprintDialogKotlin/app/src/main/res/values/strings.xml

```diff
@@ -44,7 +44,7 @@
     <string name="new_fingerprint_enrolled_description">
         A new fingerprint was added to this device, so your password is required.
     </string>
-    <string name="use_fingerprint_in_future">Use fingerprint in the future</string>
+    <string name="use_fingerprint_in_future">Use biometrics in the future</string>
```

Comment:
Should we also update the other strings from "fingerprint" to "biometrics"?

Reply:
We wanted to minimize the work that happens in this pull request to focus on what is necessary for migration. And then leave other fixes for subsequent pull requests.